sdk/attestation/azure-security-attestation/test/CMakeLists.txt

```
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
+
+project (azure-security-attestation-test LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_compile_definitions(AZURE_TEST_DATA_PATH="${CMAKE_BINARY_DIR}")
+
+# Export the test folder for recordings access.
+add_compile_definitions(AZURE_TEST_RECORDING_DIR="${CMAKE_CURRENT_LIST_DIR}")
+
+include(GoogleTest)
+
+add_executable (
+     azure-security-attestation-test
+     ut/macro_guard.cpp
+     ut/crypto_test.cpp
+     ut/attestation_test.cpp
+)
+
```

```suggestion
create_per_service_target_build(azure-security-attestation-test attestation)
```

Add this so your test can be built and run on CI

---

Added. Thanks Victor.